actor/v7action/actor.go

```diff
@@ -35,12 +44,18 @@ func NewActor(
 	routingClient RoutingClient,
 	clk clock.Clock,
 ) *Actor {
+	authActor := NewDefaultAuthActor(config, uaaClient)
+	if config != nil && config.IsCFOnK8s() {
+		authActor = NewKubernetesAuthActor(config, NewDefaultKubernetesConfigGetter())
+	}
+
 	return &Actor{
 		CloudControllerClient: client,
 		Config:                config,
```

Comment:
Do we still need the config and uaaClient fields in the Actor. It seems like we are now passing them into the AuthActor. with: @hjcalderon10